src/primitives/certificate.cpp

```diff
@@ -338,14 +338,18 @@ CFieldElement CScCertificate::GetDataHash(const Sidechain::ScFixedParameters& sc
     if (bt_list_len == 0)
         bt_list_ptr = nullptr;
 
+    wrappedFieldPtr sptrScId = CFieldElement(input.scId).GetFieldElement();
+    field_t* scidFe = sptrScId.get();
+
     wrappedFieldPtr   sptrConst  = input.constant.GetFieldElement();
     wrappedFieldPtr   sptrCum    = input.endEpochCumScTxCommTreeRoot.GetFieldElement();
     wrappedScProofPtr sptrProof  = input.certProof.GetProofPtr();
     wrappedScVkeyPtr  sptrCertVk = input.CertVk.GetVKeyPtr();
```

Comment:
the 4 items above are not used. Do we need to keep them?

Reply:
Ok, removed; also added log trace with return code value